core/server/config/overrides.json

```diff
@@ -63,9 +63,6 @@
         "cannotScheduleAPostBeforeInMinutes": 2,
         "publishAPostBySchedulerToleranceInMinutes": 2
     },
-    "theme": {
-        "timezone": "Etc/UTC"
-    },
```

Comment:
We don't need this config property at all in my opinion.
Ghost server runs in UTC, see https://github.com/TryGhost/Ghost/blob/master/core/server/overrides.js

The blog uses Etc/UTC as default for it's blog timezone, see https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/default-settings.json#L47

So the blog timezone is only stored in the database.
Maybe i am wrong, but i couldn't see any reason to keep it in config and in db settings.